src/main/java/uk/gov/register/db/RecordQueryDAO.java

```diff
@@ -27,20 +27,20 @@
     @SqlQuery("SELECT count FROM total_records")
     public abstract int getTotalRecords();
 
-    @SqlQuery("select entry_number, timestamp, e.sha256hex as sha256hex, key, content from entry e, item i where e.sha256hex=i.sha256hex and e.entry_number = (select entry_number from current_keys where current_keys.key=:key)")
+    @SqlQuery("select e.entry_number, e.timestamp, ei.sha256hex as sha256hex, e.key, i.content from entry e join entry_item ei on ei.entry_number = e.entry_number join item i on i.sha256hex = ei.sha256hex and e.entry_number = (select entry_number from current_keys where current_keys.key=:key)")
```

Comment:
These queries also only return a single item for each entry (record). They need to be changed to return all the items. This also means that you may need to change `Record` to contain a list of items, and leave the old single-item getters/constructors there for now (similar to what we did for `Entry`).